GitOut/Features/Wpf/AttachedProperties/TextZoom.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Windows;
+using System.Windows.Input;
+using System.Windows.Media;
+
+namespace GitOut.Features.Wpf.AttachedProperties
```

Comment:
AttachedProperties is a type classification rather than a feature. I believe we should use feature names in namespaces rather than types.

Reply:
> 
> 
> AttachedProperties is a type classification rather than a feature. I believe we should use feature names in namespaces rather than types.

not sure what you mean

Reply:
I mean AttachedProperty is a type of file (similar to ViewModel, Model, Controller, Service and so on) instead of a feature (as in Net, IO, Git, Zoom, Text).
Unless we consider AttachedProperties to be a specific feature the file should be placed in a different folder